templates/js-empty/main.js

```diff
@@ -1,23 +1,16 @@
 // This is the main Node.js source code file of your actor.
 // An actor is a program that takes an input and produces an output.
 
-// For more information, see https://docs.apify.com/sdk/js
+// Import Apify SDK (Read more at https://docs.apify.com/sdk/js)
 import { Actor } from 'apify';
-// For more information, see https://crawlee.dev
-// import { CheerioCrawler } from 'crawlee';
 
 // Initialize the Apify SDK
 await Actor.init();
 
-// Get input of the actor (here only for demonstration purposes).
-const input = await Actor.getInput();
-console.log('Input:');
-console.dir(input);
-
+console.log('Hello from the Actor!');
```

Comment:
This is OCD, but I think it would be nice if all the empty templates logged the same thing 😄 Feel free to ignore me though.

Reply:
Yeah, fair. We haven't synced this. I'll make sure it's the same for all the empty ones.